liferay-echo/liferay-portal
# LPS-172113 Asset Publisher configuration uses inconsistent values for select more than one selectors

Hi Team,
Can you please review this?

Motivation
=======
https://issues.liferay.com/browse/LPS-172113
AssetPublisherExportImportPortletPreferencesProcessor can't process `anyClassTypeDLFileEntryAssetRendererFactory`, `anyClassTypeJournalArticleAssetRendererFactory` and `anyAssetType` preferences, because "false" value can't be set on them.

Proposed Solution
========
Revert the "select-more-than-one" values to "false" like how "any" was reverted to "true"

Steps to Verify
========
1. In the Global site, create 3 document types (I've called them TYPE_A, TYPE_B and TYPE_C)
1. Create a new site : Alpha
1. In Alpha, create 3 documents using the 3 types we have created
1. In Alpha, create a new content page and add the Asset Publisher Widget
1. Configure the Asset Publisher Widget:
   1. Asset Selection: Dynamic
   1. Source - Asset Type: Document
   1. Source - Document Type: Select More Than One...
   1. Source - Selected: TYPE_A and TYPE_B (we don't include TYPE_C)
1. The Asset Publisher displays 2 out of the 3 documents we have created, this is expected.
1. On the Alpha site, go to Publishing - Export and make a complete export of the site.
1. Create a new site : Beta
1. On the Beta site, go to Publishing - Import and import the Alpha site's lar

**Expected behaviour:** The Beta site page displays the 2 filtered documents.
The configuration of the Asset Publisher on the Beta page is the same as on Alpha.

**Actual behaviour:** The Beta site page displays the 2 filtered documents (the one of TYPE_C is not displayed, as expected).
However, if you go to the Asset Publisher configuration, it's wrong because you have;
   1. Asset Selection: Dynamic
   1. Source - Asset Type: Document
   1. Source - Document Type: Any
   1. The filter on document type TYPE_A and TYPE_B is lost.

Cheers,
Balázs
